### PR TITLE
[REF] auto-fix conventions (+ manual fixes)

### PR DESCRIPTION
**Pull request by mdietrichc2c**

### CODE REVIEW

File: l10n_ch_sepa/base_sepa/msg_sepa.py [hunk 1]
```diff
@@ -74,6 +74,7 @@ def _is_xsd_valid(self):
 
 
 class MsgSEPAFactory(object):
+
     """This class is a factory that creates SEPA message in order to allow
```
Comment (bwrsandman) [inline l10n_ch_sepa/base_sepa/msg_sepa.py:76]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_scan_bvr/tests/test_scan_bvr.py [hunk 2]
```diff
@@ -23,6 +23,7 @@
 
 
 class test_scan_bvr(common.TransactionCase):
+
     """ Test the wizard for bvr line scanning """
```
Comment (bwrsandman) [inline l10n_ch_scan_bvr/tests/test_scan_bvr.py:26]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_payment_slip_account_statement_base_completion/statement.py [hunk 3]
```diff
@@ -27,6 +27,7 @@
 
 
 class account_statement_completion_rule(orm.Model):
+
     """ Add a rule based on BVR Reference """
```
Comment (bwrsandman) [inline l10n_ch_payment_slip_account_statement_base_completion/statement.py:30]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_payment_slip/report/multi_report_webkit_html.py [hunk 4]
```diff
@@ -30,6 +30,7 @@
 
 
 class L10nCHReportWebkitHtmlMulti(report_sxw.rml_parse):
+
     """Report that output single BVR from invoice.
     This report is deprectated and will be merged
```
Comment (bwrsandman) [inline l10n_ch_payment_slip/report/multi_report_webkit_html.py:34]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_payment_slip/invoice.py [hunk 5]
```diff
@@ -227,6 +228,7 @@ def copy(self, cr, uid, inv_id, default=None, context=None):
 
 
 class AccountTaxCode(Model):
+
     """Inherit account tax code in order
     to add a Case code"""
```
Comment (bwrsandman) [inline l10n_ch_payment_slip/invoice.py:232]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_payment_slip/invoice.py [hunk 6]
```diff
@@ -82,6 +82,7 @@ def _compute_bvr_ref(self, cr, uid, move_line, context=None):
 
 
 class AccountInvoice(Model):
+
     """Inherit account.invoice in order to add bvr
     printing functionnalites. BVR is a Swiss payment vector"""
```
Comment (bwrsandman) [inline l10n_ch_payment_slip/invoice.py:86]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_payment_slip/company.py [hunk 7]
```diff
@@ -23,6 +23,7 @@
 
 
 class ResCompany(Model):
+
     """override company in order to add bvr vertical and
```
Comment (bwrsandman) [inline l10n_ch_payment_slip/company.py:26]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_payment_slip/bank.py [hunk 8]
```diff
@@ -21,6 +21,7 @@
 
 
 class ResPartnerBank(orm.Model):
+
     """
     Inherit res.partner.bank class in order to add swiss specific fields
```
Comment (bwrsandman) [inline l10n_ch_payment_slip/bank.py:25]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_lsv_dd/model/move_line.py [hunk 9]
```diff
@@ -23,6 +23,7 @@
 
 
 class account_move_line(orm.Model):
+
     '''
     Use hooks to add bvr ref generation if account is IBAN and has LSV
     identifier
```
Comment (bwrsandman) [inline l10n_ch_lsv_dd/model/move_line.py:28]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_dta/wizard/create_dta.py [hunk 10]
```diff
@@ -484,10 +491,12 @@ def init_local_context(self):
 
 
 class record_gt890(record):
+
     """Implements Total Record of DTA file
     if behaves like a account payment order
```
Comment (bwrsandman) [inline l10n_ch_dta/wizard/create_dta.py:490]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_dta/wizard/create_dta.py [hunk 11]
```diff
@@ -371,6 +377,7 @@ def init_local_context(self):
 
 
 class record_gt836(record):
+
     """Implements iban record"""
```
Comment (bwrsandman) [inline l10n_ch_dta/wizard/create_dta.py:376]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_dta/wizard/create_dta.py [hunk 12]
```diff
@@ -286,9 +290,11 @@ def validate_global_context_dict(self):
 
 
 class record_gt827(postal_record):
+
     """
     Swiss internal (bvpost and bvbank) record implemetation
```
Comment (bwrsandman) [inline l10n_ch_dta/wizard/create_dta.py:291]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_dta/wizard/create_dta.py [hunk 13]
```diff
@@ -171,6 +174,7 @@ def validate_global_context_dict(self):
 
 
 class record_gt826(postal_record):
+
     """ BVR record implementation"""
```
Comment (bwrsandman) [inline l10n_ch_dta/wizard/create_dta.py:175]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_dta/wizard/create_dta.py [hunk 14]
```diff
@@ -139,10 +140,12 @@ def generate(self):
 
 
 class postal_record(record):
+
     """Class that propose common
     knowledge for all postal account type
```
Comment (bwrsandman) [inline l10n_ch_dta/wizard/create_dta.py:143]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_dta/wizard/create_dta.py [hunk 15]
```diff
@@ -78,6 +78,7 @@ def tr(string_in):
 
 
 class record(object):
+
     """Abstract class that provides common
     knowledge for any kind of post/bank account
```
Comment (bwrsandman) [inline l10n_ch_dta/wizard/create_dta.py:82]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_dta/bank.py [hunk 16]
```diff
@@ -21,6 +21,7 @@
 
 
 class ResPartnerBank(orm.Model):
+
     """
     Inherit res.partner.bank class in order to add swiss specific fields
```
Comment (bwrsandman) [inline l10n_ch_dta/bank.py:25]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_credit_control_payment_slip_report/credit_control_printer.py [hunk 17]
```diff
@@ -23,6 +23,7 @@
 
 
 class CreditControlPrinter(orm.TransientModel):
+
     """Print lines"""
     _inherit = 'credit.control.printer'
```
Comment (bwrsandman) [inline l10n_ch_credit_control_payment_slip_report/credit_control_printer.py:28]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_credit_control_payment_slip_report/credit_control_communication.py [hunk 18]
```diff
@@ -37,12 +38,13 @@ def _generate_report_bvr(self, cr, uid, line_ids, context=None):
...
+        result, bvr_format = service.create(cr, uid, line_ids, {}, {})
         return result
 
 
 class MultiBvrWebKitParserCreditControl(
         multi_report_webkit_html.L10nCHReportWebkitHtmlMulti):
+
     """We define a new parser because this report take move line
        In parameter insted of an invoice, so the function get_obj_reference
        return directly ids"""
```
Comment (bwrsandman) [inline l10n_ch_credit_control_payment_slip_report/credit_control_communication.py:49]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_credit_control_payment_slip_report/credit_control_communication.py [hunk 19]
```diff
@@ -25,6 +25,7 @@
 
 
 class CreditCommunication(orm.TransientModel):
+
     """Shell class used to provide a base model to email template
        and reporting.
```
Comment (bwrsandman) [inline l10n_ch_credit_control_payment_slip_report/credit_control_communication.py:29]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_credit_control_payment_slip_report/account.py [hunk 20]
```diff
@@ -22,6 +22,7 @@
 
 
 class account_move_line(orm.Model):
+
     """Overrride BVR amount to take in account dunning fees"""
```
Comment (bwrsandman) [inline l10n_ch_credit_control_payment_slip_report/account.py:25]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_base_bank/bank.py [hunk 21]
```diff
@@ -144,6 +145,7 @@ def name_search(self, cursor, uid, name, args=None, operator='ilike',
 
 
 class ResPartnerBank(orm.Model, BankCommon):
+
     """
     Inherit res.partner.bank class in order to add swiss specific fields
     and state controls
```
Comment (bwrsandman) [inline l10n_ch_base_bank/bank.py:149]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_base_bank/bank.py [hunk 22]
```diff
@@ -53,6 +53,7 @@ def _check_5_pos_postal_num(self, number):
 
 
 class Bank(orm.Model, BankCommon):
+
     """Inherit res.bank class in order to add swiss specific field"""
```
Comment (bwrsandman) [inline l10n_ch_base_bank/bank.py:56]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_account_statement_base_import/parser/v11_file_parser.py [hunk 23]
```diff
@@ -32,6 +32,7 @@ def is_total_record_type4(test):
 
 
 class V11FileParser(BankStatementImportParser):
+
     """
     A v11 file parser.
```
Comment (bwrsandman) [inline l10n_ch_account_statement_base_import/parser/v11_file_parser.py:36]: :-1: Is the newline before a doctring a positive change to style?

File: l10n_ch_account_statement_base_import/parser/postfinance_file_parser.py [hunk 24]
```diff
@@ -32,9 +32,11 @@
 
 
 class XMLPFParser(BankStatementImportParser):
+
     """
     Parser for XML Postfinance Statements (can be wrapped in a tar.gz file)
     """
```
Comment (bwrsandman) [inline l10n_ch_account_statement_base_import/parser/postfinance_file_parser.py:37]: :-1: Is the newline before a doctring a positive change to style?

**Comment by gurneyalex** [comment]
:+1: with my fixes applied

**Comment by lepistone** [comment]
:+1:

**Comment by nbessi** [comment]
:+1:

**Pull request merged**